test/libmpv_encode.c

```diff
@@ -115,9 +113,17 @@ int main(int argc, char *argv[])
         return 1;
 
     static char path[] = "./testout.XXXXXX";
+
+#ifdef _WIN32
     out_path = mktemp(path);
```

Comment:
If it is only windows, consider using `_mktemp`

Reply:
Sure if that's more microsoft approved™.